src/foam/u2/DAOChoicesListView.js

```diff
@@ -28,51 +26,48 @@ foam.CLASS({
   properties: [
     'of',
     {
-      name: 'sections',
+      name: 'choices',
       value: []
```

Comment:
Don't do value: [] or different instances will share the same array. Instead do factory: function() { return []; }